packages/knip/test/plugins/webpack.test.ts

```diff
@@ -26,7 +26,7 @@ test('Find dependencies with the Webpack plugin', async () => {
     devDependencies: 2,
     unlisted: 2,
     unresolved: 1,
-    processed: 12,
-    total: 12,
+    processed: 13,
+    total: 13,
```

Comment:
+1 because the new file `my-loader.js` now gets correctly processed; there is no additional unused file, which means it gets seen correctly.